ml/ml_table.yaml

```diff
@@ -48,6 +48,16 @@ datasets:
     required: false
     doc: The class probabilities from which these class labels were determined
 
+- data_type_def: TopKProbabilities
+  data_type_inc: SupervisedOutput
+  dtype: float
+  doc: A column for storing the top k class probabilities. Probabilities should be ordered in descending order.
+
+- data_type_def: TopKClasses
+  data_type_inc: SupervisedOutput
+  dtype: float
+  doc: A column for storing the top k classes
```

Comment:
Can you give an example of data for this column? Why is it a float?

Reply:
Should TopKProbabilities and TopKClasses always be present together? And should len(TopKProbabilities) == len(TopKClasses)

Reply:
TopKClasses should be an int. I will fix that. 

They don't always need to be stored together.

Reply:
For context, top-k classes gets used for top-k accuracy calculation. Checkout the [scikit-learn function](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.top_k_accuracy_score.html) for a more detailed explanation